nncf/layer_utils.py

```diff
@@ -79,3 +80,37 @@ def forward(self, *args):
             if op_results is not None:
                 results = op_results
         return results
+
+
+class CompressionParameter(nn.Parameter):
+    """
+    The class that should be used in all compression algorithms instead of torch.nn.Parameter.
+
+    This class utilize `compression_lr_multiplier` parameter from :class:`nncf.NNCFConfig`
+    to increase/decrease gradients for compression algorithms' parameters.
+
+    N.B. If you want to set `requires_grad` parameter, please only use `requires_grad_` method.
+    """
+
+    def __new__(cls, data: torch.Tensor = None, requires_grad: bool = True, compression_lr_multiplier: float = None):
+        return super().__new__(cls, data, requires_grad=requires_grad)
+
+    def __init__(self, data: torch.Tensor = None, requires_grad: bool = True, compression_lr_multiplier: float = None):
```

Comment:
```suggestion
    def __init__(self, data: torch.Tensor = None, requires_grad: bool = True, compression_lr_multiplier: float = 1.0):
```

Reply:
If `compression_lr_multiplier` is None the hook will not be applied. But in the case when `compression_lr_multiplier`=1.0 there will floating-point multiplication which could lead to accumulating differences in calculation.